css/app-sidebar.scss

```diff
@@ -899,8 +899,14 @@
 	align-items: center;
 	width: 100%;
 
+	&--rtl{
+		text-align: right;
+	}
 	// Account for avatar width (because it is position: relative)
-	padding-right: 32px;
+	&--ltr{
+		padding-right: 32px;
```

Comment:
Why not just `padding-block-end`?

Reply:
<img width="454" alt="image" src="https://github.com/user-attachments/assets/6be01d5e-741b-4417-95c7-08d2a9c3a877">

Reply:
margin-inline-start works i think